src/controllers/queries/__tests__/index.test.ts

```diff
@@ -135,12 +154,17 @@ describe('QueriesController', () => {
           quantity: 111,
         })
         .then(toHTMLString)
+      expect(dbMock.update.getCall(0).args).to.deep.equal([
+        'query',
+        { id: 'ccaaaaaa-0000-0000-0000-d8ae0805059e' },
+        { status: 'errored' },
+      ])
 
       expect(result).to.equal('scope3_error_scope3')
     })
   })
 
-  describe('query responses', () => {
+  describe.only('query responses', () => {
```

Comment:
remove only

Reply:
Ooops, this should have been update, added some unit tests for DRPC events as well, but well, they could be better.